### ISSUE TITLE
Is it now possible to launch calabash in parallel on multiple physical devices?

### ISSUE THREAD

**Issue by gbrhaz** [calabash/calabash-ios#1303]
Previously I know it wasn't possible to do this as Instruments process could only be run once; however, I've been reading various things about how it is now possible in iOS 10/Xcode 8. Is this true? Are there any specific things I would need to do to achieve this?

**Comment by xv-chris-w** [comment]
yes, it is possible with some 3rd party library,  calabash ios does not support by default. 

https://github.com/rajdeepv/parallel_calabash
